Tests/Application/Kernel.php

```diff
@@ -37,4 +37,15 @@ public function registerContainerConfiguration(LoaderInterface $loader): void
 
         $loader->load(__DIR__ . '/config/config_' . $this->getContext() . '.yml');
     }
+
+    protected function getKernelParameters(): array
+    {
+        $parameters = parent::getKernelParameters();
+
+        $gedmoReflection = new \ReflectionClass(\Gedmo\Exception::class);
+        $parameters['gedmo_directory'] = \dirname($gedmoReflection->getFileName());
```

Comment:
gedmo prefer lowest fix we use already in other bundles

Reply:
again removed as sulu needs to be increased to a version which already contains this fix